dydx-testnet-4/gentx/gentx-StakeRun.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"StakeRun","identity":"6EAA935D0E26A462","website":"https://stake.testnet.run","security_contact":"","details":"Best uptime, securely and non-custodial staking. Professional staking service, easily stake your assets with StakeRun"},"commission":{"rate":"0.050000000000000000","max_rate":"0.100000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1arrlapn93ysd3hle9z4apsm55l0qxzflvxqrmy","validator_address":"dydxvaloper1arrlapn93ysd3hle9z4apsm55l0qxzflfck4s4","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"YHIJubyTUk32ULwyZ6rJiD37mpgRA63whCFJjzvvbZs="},"value":{"denom":"dv4tnt","amount":"50000000000"}}],"memo":"de3cd278c3795323beaf8f2bb07d365857e51261@88.99.143.105:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A79zCdiqWw8bXKupxPpDaBBXiDtHADUtgLeLSP8Yz0Ps"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["fWj9f9gWwxt1hFuMBPjX0uIWWriPj58bcLcjZ4y7OgYOpX7gLgDHpBnwKw59URdZtCK7ReI1z/k1r8izYnTcEw=="]}
```

Comment:
@Errorist79 the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is adv4tnt and the self-delegation amount is 500000000000000000000000adv4tnt

Reply:
@2dpetkov thanks for the heads up, it fixed and could you check  again please?